account_move_line_manufacture_info/models/stock_move.py

```diff
@@ -0,0 +1,25 @@
+# © 2019 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class StockMove(models.Model):
+    _inherit = 'stock.move'
+
+    @api.model
+    def _prepare_account_move_line(self, qty, cost,
```

Comment:
@ageficent Maybe a little test on that ?

Reply:
Done :+1: